test/test_nestedtensor.py

```diff
@@ -3876,6 +3876,14 @@ def test_is_contiguous(self, device):
         self.assertTrue(not nt_noncontiguous.is_contiguous(memory_format=torch.contiguous_format))
         self.assertTrue(nt_contiguous_narrow.is_contiguous(memory_format=torch.contiguous_format))
 
+    def test_layout_under_torch_dispatch_mode():
```

Comment:
```suggestion
    def test_layout_under_torch_dispatch_mode(self):
```

I think you need this for the test to run, no?